doc/guides/dts/index.rst

```diff
@@ -457,6 +457,50 @@ hardware resources:
       uart-1 = &uart1;
    };
 
+To have a generic name in various Zephyr sample and test code we define some
+common aliases.
+
+The full set of Zephyr-specific ``aliases`` properties follows:
+
+.. list-table::
+   :header-rows: 1
+
+   * - ``aliases`` property name
+     - Generated symbol
+
+   * - ``led0``
+     - a ``gpio-leds`` compatiable LED device - instance 0
```

Comment:
*compatible

Maybe "`gpio-leds`-compatible", since it's an adjective.

(and similarly for the other ones).

Reply:
Might be nice to clarify e.g. above the table that this means compatible as in `compatible = ...`/`compatible: ...` too.